pom.xml

```diff
@@ -172,7 +172,7 @@
         <zstd-jni.version>1.5.6-3</zstd-jni.version><!-- @sync org.apache.kafka:kafka-clients:${kafka.version} dep:com.github.luben:zstd-jni -->
 
         <!-- Test dependency versions (keep sorted alphabetically) -->
-        <certificate.generator.version>0.7.1</certificate.generator.version>
+        <smallrye-certificate-generator-junit5.version>0.8.1</smallrye-certificate-generator-junit5.version>
```

Comment:
```suggestion
        <smallrye-certificate-generator-junit5.version>0.8.1</smallrye-certificate-generator-junit5.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:io.smallrye.certs:smallrye-certificate-generator -->
```
To be sure to stay in sync with Quarkus BOM. See also https://github.com/quarkusio/quarkus/pull/43393

Reply:
Ah, thanks. I missed it was already part of Quarkus 3.15.0.

I'll add the sync bits.

Reply:
Latest commit adds `@sync`.